packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTree.tsx

```diff
@@ -215,3 +216,16 @@ const isFilteredDataProvider = (dataProvider: IPresentationTreeDataProvider | IF
 const getFilteredDataProvider = (dataProvider: IPresentationTreeDataProvider | IFilteredPresentationTreeDataProvider): IFilteredPresentationTreeDataProvider | undefined => {
   return isFilteredDataProvider(dataProvider) ? dataProvider : undefined;
 };
+
+const useModelsTreeRenderer = () => {
+  const nodeRenderer = (props: TreeNodeRendererProps) => (
+    createVisibilityTreeNodeRenderer(true, false)({ ...props, className: props.node.item.parentId === undefined ? "disable-expander": undefined })
+  );
+
+  return (props: TreeRendererProps) => ( // eslint-disable-line react/display-name
```

Comment:
```suggestion
  return function ModelsTreeRenderer(props: TreeRendererProps) {
```

I think something like this should work to avoid eslint error